test/Makefile

```diff
@@ -16,6 +16,9 @@ ADDITIONAL_CPPFLAGS += -Wall -Wextra
 include $(FRAMEWORK_DIR)/build.mk
 include $(FRAMEWORK_DIR)/moose.mk
 
+ADDITIONAL_LIBS := "-L/opt/moose/libpng-1.6.25/lib -lz -lpng16"
+ADDITIONAL_INCLUDES := "-I/opt/moose/libpng-1.6.25/include/libpng16"
```

Comment:
This is the right idea but we can't hardcode these paths. I can work on this for you.

Reply:
Make sure you delete these lines!